mergify_engine/context.py

```diff
@@ -426,18 +426,31 @@ async def get_mergify_config_file(self) -> typing.Optional[MergifyConfigFile]:
         config_location_cache = self.get_config_location_cache_key(
             self.installation.owner_login, self.repo["name"]
         )
+        config_file_content_key = self._config_file_content_cache_key(
+            self.installation.owner_id, self.repo["id"]
+        )
         cached_filename = await self.installation.redis.get(config_location_cache)
+        cached_config_file_content = await self.installation.redis.get(
+            config_file_content_key
+        )
         async for config_file in self.iter_mergify_config_files(
             preferred_filename=cached_filename
         ):
             if cached_filename != config_file["path"]:
                 await self.installation.redis.set(
                     config_location_cache, config_file["path"], ex=60 * 60 * 24 * 31
                 )
+
+            if cached_config_file_content != config_file["content"]:
+                await self.installation.redis.set(
```

Comment:
I feel this should be in a pipe?

Reply:
performance wise it would be better indeed !